client/app/components/routes/operations/Page.tsx

```diff
@@ -3,14 +3,30 @@ import { Button } from "@mui/material";
 import { Suspense } from "react";
 import Operations from "@/app/components/routes/operations/Operations";
 import Loading from "@/app/components/loading/SkeletonGrid";
+import { actionHandler, getToken } from "@/app/utils/actions";
+
+export default async function OperationsPage() {
+  // 👤 Use NextAuth.js encrypted token to get authenticated user's user_guid
+  const token = await getToken();
+  const uid = token?.user_guid ?? "";
+
+  // 🚀 API call: Get operator id associated with this user_guid
+  const response = await actionHandler(
+    `registration/user-operator-id/${uid}`,
+    "GET",
+    "",
+  );
+  const operator_id = response?.operator_id ?? "";
 
-export default function OperationsPage() {
   return (
     <>
       <h1>Operations List</h1>
-      <Link href="/dashboard/operations/0/1">
-        <Button variant="contained">Add Operation</Button>
-      </Link>
+      {/* Conditionally render the button only if operator_id is an integer */}
+      {typeof operator_id === "number" && Number.isInteger(operator_id) && (
```

Comment:
If reporters aren't allowed to add operations (I'm not sure, maybe the designers know), then this will need another condition besides just the existence of an operator id.

Sidenote, I'm getting confused by the different roles. The UserOperator model has ADMIN and REPORTER, and then we have the app roles as well (cas_pending, industry_user_admin, etc.). If (?) industry_user=REPORTER and industry_user_admin=ADMIN, then maybe we can use app role to determine whether or not to show the button